nlgeval/tests/test_nlgeval.py

```diff
@@ -160,3 +190,11 @@ def test_compute_metrics(self):
         self.assertAlmostEqual(0.568696, scores['VectorExtremaCosineSimilarity'], places=5)
         self.assertAlmostEqual(0.784205, scores['GreedyMatchingScore'], places=5)
         self.assertEqual(12, len(scores))
+
+    def test_compute_metricss(self):
+        # The example from the README.
+        root_dir = os.path.join(os.path.dirname(__file__), '..', '..')
+        hypothesis = os.path.join(root_dir, 'examples/6.csv')
+        references = os.path.join(root_dir, 'examples/7.pt_beam_1.csv')
+        scores = nlgeval.compute_metrics(hypothesis, [references])
+        print(scores)
```

Comment:
Same, better to assert the result you expect.

Reply:
I found that this case is not necessary. I will remove it